lib/lte_link_control/modules/Kconfig

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config LTE_LC_CONEVAL
+	bool ""
+	default n
+
+config LTE_LC_EDRX
+	bool ""
+	default n
+
+config LTE_LC_NCELLMEAS
+	bool ""
+	default n
+
+config LTE_LC_PERIODICSEARCHCONF
+	bool ""
+	default n
+
+config LTE_LC_PSM
+	bool ""
+	default n
+
+config LTE_LC_RAI
+	bool ""
+	default n
+
+config LTE_LC_REDMOB
+	bool ""
+	default n
+
+config LTE_LC_XFACTORYRESET
+	bool ""
+	default n
+
+config LTE_LC_XMODEMSLEEP
+	bool ""
+	default n
+
+config LTE_LC_XT3412
```

Comment:
The naming here is something I'm thinking about. This is for sending `LTE_LC_EVT_TAU_PRE_WARNING` and mapping here is not intuitive. That is, how do you know to enable `CONFIG_LTE_LC_XT3412` to be able to received those events?

Reply:
The names are modeled after the respective AT commands, but this can always be changed of course. Are there other modules that you think should be renamed?

Reply:
I think others are self-explanatory. I'd like to keep AT command naming for the implementation but this XT3412 is something we would need to document for clients somehow if we don't rename it.

Reply:
Agree